__tests__/data/june.json

```diff
@@ -0,0 +1,323 @@
+[
+  {
+    "description": "Identify call without userId",
```

Comment:
to remind, we need to add this info in doc. If not mentioned by now

Reply:
userId is required field. Already, mentioned this in research spec.